config/config.go

```diff
@@ -189,6 +194,15 @@ type elasticsearch struct {
 	URL string
 }
 
+// balancer contains fields for the load balancer.
+type balancer struct {
+	// Addr is the public facing address for the balancer.
+	Addr string
+	// Fallback is the stream to default to.
+	Fallback string
+	Relays   []*radio.Relay
```

Comment:
the configuration struct shouldn't be mutable, these relays would be

Reply:
This is no longer a problem, as we load relays from the DB now.